docker

```diff
@@ -57,3 +57,15 @@ docker login
 
 # To push a docker image into dockerhub repository:
 docker push <image-name>:<image-tag-name>
+
+# To list all networks daemon knows
+docker network ls
```

Comment:
```suggestion
# List all networks daemon knows about:
docker network ls
```

- Optional change: `to` is not mandatory.
- Optional change: `a specific` may sound better.
- Optional change: maybe it's better to use a trailing comma like in TlDr pages.